src/app_model/types/_keys/_keybindings.py

```diff
@@ -210,6 +246,24 @@ def to_int(self, os: Optional[OperatingSystem] = None) -> int:
             return KeyChord(*parts)
         return parts[0]
 
+    def to_text(
+        self,
+        os: Optional[OperatingSystem] = None,
+        use_symbols: bool = False,
+        joinchar: str = "+",
+    ) -> str:
+        """Get a text representation of this KeyBinding.
```

Comment:
i think we need a slight additional clarification on when this would be used with `symbols==False` instead of just using `__str__`.  Currently, it reads like the os-specificity is *only* for the symbols. (same with other docstring)

And, since we're adding this, i think we should perhaps also add a docstring to `__str__` clarifying that it returns a normalized string that is constant across all OSes